usecase/src/main/java/com/fiap/burger/usecase/usecase/DefaultPaymentUseCase.java

```diff
@@ -36,9 +37,9 @@ public List<Payment> findByOrderId(Long orderId) {
 
     public Payment insert(Long orderId) {
 
-        Order order = orderUseCase.findById(orderId);
+        Order order = orderGateway.findById(orderId);
 
-        if (!orderUseCase.canBePaid(order.getStatus())) {
+        if (!orderGateway.canBePaid(order.getStatus())) {
```

Comment:
Como a função `order.canBePaid` já existe, não precisa de um método canBePaid em `orderGateway`.
```suggestion
        if (!order.canBePaid()) {
```